yarn.lock

```diff
@@ -3253,6 +3316,15 @@ postcss@^8.3.11, postcss@^8.4.5:
     picocolors "^1.0.0"
     source-map-js "^1.0.1"
 
+postcss@^8.4.14:
```

Comment:
Why these are added? Don't we already have this dep?

Reply:
Agree, that lock file changes look weird..

Reply:
Agree, that lock file changes look weird..

Reply:
should be deduped now